src/insert/spacefinder/article.ts

```diff
@@ -143,47 +100,6 @@ const decideAdditionalSizes = async (
 };
 
 const addDesktopInline1 = (fillSlot: FillAdSlot): Promise<boolean> => {
-	const tweakpoint = getCurrentTweakpoint();
-	const hasLeftCol = ['leftCol', 'wide'].includes(tweakpoint);
-
-	let ignoreList = `:scope > :not(p):not(h2):not(ul):not(.${adSlotContainerClass}):not(#sign-in-gate):not(.sfdebug)`;
-	if (hasLeftCol) {
-		ignoreList +=
-			':not([data-spacefinder-role="richLink"]):not([data-spacefinder-role="thumbnail"])';
-	}
-
-	const isImmersive = window.guardian.config.page.isImmersive;
-
-	const rules: SpacefinderRules = {
-		bodySelector: articleBodySelector,
-		candidateSelector: ':scope > p, [data-spacefinder-role="nested"] > p',
-		minAbove: isImmersive ? 700 : 300,
-		minBelow: 300,
-		opponentSelectorRules: {
-			// don't place ads right after a heading
-			':scope > h2, [data-spacefinder-role="nested"] > h2': {
-				minAboveSlot: isInHighValueSection ? 150 : 5,
-				minBelowSlot: isInHighValueSection ? 0 : 190,
-			},
-			[`.${adSlotContainerClass}`]: {
-				minAboveSlot: 500,
-				minBelowSlot: 500,
-			},
-			[ignoreList]: {
-				minAboveSlot: 35,
-				minBelowSlot: 400,
-			},
-			'[data-spacefinder-role="immersive"]': {
```

Comment:
Why is there a different `minAbove` and minBelow` for immersive in the refactor? I believe it called `rightColumnOpponentSelector` in `rules.ts` which has `minAboveSlot: 35` and `minBelowSlot: 400`.

Reply:
I think this rule is actually a mistake, there's no reason for inline1 to be 600px from immersive elements! 400 also seems excessive but safer I think.